x-pack/legacy/plugins/code/public/components/commits/commit.tsx

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import {
+  EuiButtonIcon,
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiPanel,
+  EuiText,
+  EuiTextColor,
+  EuiCopy,
+  EuiTitle,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { CommitInfo } from '../../../model/commit';
+import { MAIN_ROOT } from '../routes';
+import { PathTypes } from '../../common/types';
+import { parseCommitMessage } from '../../../common/commit_utils';
+
+const COMMIT_ID_LENGTH = 8;
+
+interface ActionProps {
+  repoUri: string;
+  commitId: string;
+}
+
+const copyLabel = i18n.translate('xpack.code.commits.copyCommitAriaLabel', {
+  defaultMessage: 'Copy the full commit ID',
```

Comment:
@mw-ding one more question: who writes/reviews this copy?

Reply:
Do you mean the default English version here or the japanese and chinese ones? The first one should be ourselves. The latter ones my understanding is the kibana ui team to handle this.

Reply:
Yeah, I was referring to the default english. I saw the guidelines and nothing looked out of line, but I wasn't sure if there was more process involved and wanted to call it out.

Reply:
mengwei is right, we need to be responsible for the English version.

also, in Code team, we do Chinese translation (at least for the first round)  by ourselves